dmutils/s3.py

```diff
@@ -99,12 +112,16 @@ def _format_key(self, key):
         :return:    dict
         """
         filename, ext = os.path.splitext(os.path.basename(key.name))
+        timestamp = None
+        if load_timestamps:
+            key = self.bucket.get_key(key.name)
```

Comment:
Why do we need to call `.get_key` when we already have a `Key` instance? Are objects returned by `bucket.list` different from `.get_key` ones?